### PR TITLE
#68 multilang markup

### LINKED ISSUE #68: [BUG] Multi language markdown files - not properly marked up
## Required information

**Describe the bug**
Markdown files that are multi language don't have proper markup when language changes from English to French. 

All French text should be wrapped in a `<div lang="fr">` because Github's pages are declared as `lang="en"`. 

**To Reproduce**

View source:

```
<html lang="en" data-color-mode="auto" data-light-theme="light" data-dark-theme="dark">
```

**Expected behaviour**
When a screen reader reads the French text, it should read it in French.

**Who shall do the work?**
I'll leave it open for anyone to take.

## Additional information (optional)

**Additional context**
 
I've opened this issue with Github to try to create a markdown way of switching languages on `.md` files using Markdown.

[Multi language page in GFM #243](https://github.com/github/cmark-gfm/issues/243)

### PR DESCRIPTION
**Pull request by csps-snap**
## markdown pages language tagging

markdown pages need some french lang attribute so that they're being read properly.

**Related issues / Requêtes associées**
closes #68

### CODE REVIEW

**Review by shawnthompson — APPROVED** [review]

**Comment by csps-snap** [comment]
huzzah

**Review by duboisp — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
+<div lang="fr">
+
 ([Français](#boîte-à-outils-d'accessibilité-numérique))
 
+</div>
+<div lang="en">
```
Comment (shawnthompson) [inline README.md:6]: remove `<div lang="en">`

File: README.md [hunk 2]
```diff
@@ -46,7 +51,9 @@ Unless otherwise noted, the source code of this project is covered under Crown C
 
 The Canada wordmark and related graphics associated with this distribution are protected under trademark law and copyright law. No permission is granted to use them outside the parameters of the Government of Canada's corporate identity program. For more information, see [Federal identity requirements](https://www.canada.ca/en/treasury-board-secretariat/topics/government-communications/federal-identity-requirements.html).
 
+</div>
```
Comment (shawnthompson) [inline README.md:54]: remove this line

**Review by shawnthompson — CHANGES_REQUESTED** [review]
Please make the changes

File: README.md (hunk 2 above)
Comment (csps-snap) [reply README.md]: lines removed

**Review by shawnthompson — APPROVED** [review]

**Pull request merged**